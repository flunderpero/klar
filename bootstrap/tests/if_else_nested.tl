let word = "Hello"

if word == "Hello" then
    print("OK: Level 1")
    if word == "there" then
        print("ERR: Level 2")
        if word == "Hello" then
            print("ERR: Level 2 nested if")
        end
    else
        print("OK: Level 2")
        if "Hello" == "Hello" then
            print("OK: Level 3.1")
        else
            print("ERR: Level 3.1")
        end
        if "Hello" == "empty if body" then
        else
            print("OK: Level 3.2")
        end
    end
else
    print("ERR: Level 1")
end
