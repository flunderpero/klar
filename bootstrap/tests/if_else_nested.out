OK: Level 1
OK: Level 2
OK: Level 3.1
OK: Level 3.2
